### ISSUE TITLE
QT UI: Game deletion not working correctly

### ISSUE THREAD

**Issue by ghost** [shadps4-emu/shadPS4#1807]
If a GAME UPDATE is deleted from the game list via the ‘Delete.../Delete Update’ menu, the game is removed from the list. Reloading the list will bring it back, but the game should not actually be removed because only the update was deleted. If a GAME UPDATE is deleted within the game list, the game itself must remain in the list.

In addition, the list content is no longer correct after deleting a GAME or GAME UPDATE. The list display is correct but unusable. If a game in the list below the deleted entry is to be started, either an error is displayed (possibly because an attempt is made to start the deleted game) or an incorrect game is started. It looks as if the view has been updated, but the list still has the old content.

**Comment by Hermiten** [comment]
Maybe @rainmakerv3 or @DanielSvoboda ? :)

**Comment by DanielSvoboda** [comment]
I also noticed that when deleting via the 'list' it works. However, if I delete the game in 'grid' mode it is not being removed from the grid after deleting.

**Comment by ghost** [comment]
? I use the list. 
Not sure whether i explained it correctly: The only problem is the when delete an UPDATE the game is removed from the list (reloading the list brings it back) but the game should only be removed from the list when i delete a GAME ...not just an update.

**Comment by DanielSvoboda** [comment]
yes, I am reporting another problem. When the game is removed and is in 'Grid View' mode, it is not being removed from the grid, only if it is reloaded. Now about this UPDATE part, I don't know how it was done...

**Comment by ghost** [comment]
Ah... sorry, didn't realize that.

**Comment by ElBread3** [comment]
Ah we're just unconditionally deleting the entry regardless of the option chosen. I can PR the fix for this as I was going to add a Delete Save Data option, but I'm going to see a movie this afternoon so it will have to wait a bit

**Comment by Hermiten** [comment]
Any update @ElBread3 ?

**Comment by Hermiten** [comment]
Fixed by https://github.com/shadps4-emu/shadPS4/pull/2261

**Comment by ghost** [comment]
The problem has been fixed, but after deleting an update PKG, the PKG version in the list is not adjusted ... a tiny flaw.

Note that If an update PKG is installed, the PKG version is adjusted.

**Comment by Hermiten** [comment]
Good to know, we can maybe open an issue to list all small fixes to do on the GUI like this one ?

**Comment by ghost** [comment]
That's fine with me, the main problem has been solved.
